modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Checkbox/Checkbox.tsx

```diff
@@ -144,6 +146,7 @@ const Main: React.FC<IProps> = ({
 };
 
 interface IProps extends ICheckboxProps {
+	defaultLanguageId?: any;
```

Comment:
Hi @georgel-pop-lr, we avoid using `any` as a type. Please, replace this line with:

```suggestion
	defaultLanguageId?: Liferay.Language.Locale;
```

Thanks! :blush: